lib/facebook/format-message.js

```diff
@@ -589,31 +545,97 @@ class receipt{
     if (!isNumber(total))
       throw new Error('Total amount must be a number');
 
-    this.output.summary.total_cost = total;
+    this.template.attachment.payload.summary.total_cost = total;
 
     return this;
   }
 
   get() {
-    if (!this.output.elements.length)
+    if (!this.template.attachment.payload.elements.length)
       throw new Error('At least one element/item is required');
 
-    if (!this.output.summary.total_cost)
+    if (!this.template.attachment.payload.summary.total_cost)
       throw new Error('Total amount is required');
 
-    let payload = this.output;
-    payload.template_type = 'receipt';
+    return this.template;
+  }
+}
 
-    return {
-      attachment: {
-        type: 'template',
-        payload: payload
-      }
-    };
+/* Deprecated methods */
+class text extends Text {
```

Comment:
Ok, but we are not using JSDoc everywhere anyway :D
I can remove the comment, but it's just a note for the next major version